.github/workflows/main.yml

```diff
@@ -19,6 +20,51 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  run-matlab-examples:
+    name: Run the MATLAB examples
+    runs-on: ubuntu-18.04
```

Comment:
This runner is deprecated, is subject to routine brownouts to notify of the pending removal, and will be removed by the end of the year. Can you switch to at least `ubuntu-20.04`, if not `ubuntu-22.04`?